rust/tests/dataframe_test.rs

```diff
@@ -1,89 +1,83 @@
-extern crate arrow;
-extern crate deltalake;
 #[cfg(feature = "rust-dataframe-ext")]
-extern crate rust_dataframe;
+mod rust_dataframe {
```

Comment:
no change in test, just moved code into its own mod to make conditional compilation annotation cleaner.

Reply:
don't think it matters much but `#![cfg(feature = "rust-dataframe-ext")]` might also work?

As a side note I was wondering if this still needs to / should be supported? It seems the repository for rust_dataframe has been archived, or did just move somewhere else?